srcpkgs/swww/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'swww'
+pkgname=swww
+version=0.9.5
+revision=1
+build_style=cargo
+hostmakedepends="scdoc pkg-config"
+makedepends="liblz4-devel"
+short_desc="Solution to your Wayland Wallpaper Woes"
+maintainer="Daniel Martinez <danielmartinez@cock.li>"
+license="GPL-3.0-only"
+homepage="https://github.com/LGFae/swww"
+changelog="https://raw.githubusercontent.com/LGFae/swww/v${version}/CHANGELOG.md"
```

Comment:
use default branch for changelog, not versioned

Reply:
done